### PR TITLE
[JENKINS-34411] Horizontal scrolling and scrolling overflows

### PR DESCRIPTION
**Pull request by andresrc**
See [JENKINS-34411](https://issues.jenkins-ci.org/browse/JENKINS-34411).

@reviewbybees (esp. @tfennelly and @recena )

### CODE REVIEW

**Comment by olivergondza** [comment]
LGTM

**Comment by recena** [comment]
@andresrc Did you try the strategy that I suggested you? (in our offline meeting)

**Comment by andresrc** [comment]
@recena I started, but I saw I needed to deal with the horizontal scrolling in any case so I did not continue that way.

**Comment by recena** [comment]
@andresrc I don't want to block your PR but I'd like to try (deepen) in that approach.

:+1:

**Comment by armfergom** [comment]
LGTM, 🐝

**Comment by andresrc** [comment]
@reviewbybees done

**Comment by andresrc** [comment]
Also problems in elements with internal viewports (such as the credentials dialog in `CreateSlaveTest#newSlave`, so I tried to implement the approach suggested by @recena 

(the horizontal scrolling part is still needed)

File: src/main/resources/org/jenkinsci/test/acceptance/selenium/overflow.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+style.type = 'text/css';
+style.id = 'ath-overflow-override'
+if (style.styleSheet){
+  style.styleSheet.cssText = css;
+} else {
+  style.appendChild(document.createTextNode(css));
+}
+
+head.appendChild(style);
```
Comment (recena) [inline src/main/resources/org/jenkinsci/test/acceptance/selenium/overflow.js]: The order is important. In this case would be recommendable to add it at the end of the rest of CSS files.

File: src/main/resources/org/jenkinsci/test/acceptance/selenium/overflow.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+var css = '* { overflow: visible !important }',
```
Comment (recena) [inline src/main/resources/org/jenkinsci/test/acceptance/selenium/overflow.js]: Better `html body *`, no?

Comment (andresrc) [inline src/main/resources/org/jenkinsci/test/acceptance/selenium/overflow.js]: addressed.

**Comment by abayer** [comment]
:bee:, merging.

**Pull request merged**